src/pages/LogIn/LogInPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import clsx from "clsx";
+
+import Button from "../../Components/Button";
+import Tittle from "../../Components/Tittle";
+import Input from "../../Components/Input";
+import { ToastLogIn } from "../../Components/Toast";
+
+import { checkUser ,checkIfCurrentNickNameExist} from "../../redux/store/usersReducer";
+
+import style from "./LogIn.module.scss";
+import "react-toastify/dist/ReactToastify.css";
+
+
+const LogInPage = () => {
+  const dispatch = useDispatch();
+  const [dataToLogIn, setDataToLogIn] = useState({});
+  const [wrongValue, setWrongValue] = useState(true);
```

Comment:
`wrongValue` bad naming. Wrong value it is not the same as empty, or alreay exist e.t.c